content/general/concepts/multi-factor-authentication/multi-factor-authentication.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: 'Multi-factor Authentication'
+Description: 'Multi-factor Authentication (MFA) is the use of more than one factor to prove the identity of a user.'
+Subjects:
+  - 'Computer Science'
+  - 'Cybersecurity'
+Tags:
+  - 'Best Practices'
+  - 'Cybersecurity'
+CatalogContent:
+  - 'paths/code-foundations'
```

Comment:
```suggestion
  - 'introduction-to-cybersecurity'
```

I would add at least one free course/path if possible. This one seemed like the most relevant.